brightener.cpp

```diff
@@ -1,36 +1,26 @@
 #include "brightener.h"
 
-ImageBrightener::ImageBrightener(std::unique_ptr<Image> inputImage) : m_inputImage(std::move(inputImage)) {
-}
-
-bool ImageBrightener::ValidateImage()
-{
-	if (m_inputImage->m_columns <= 1024 && m_inputImage->m_rows <= 1024) {
-		return true;
-	}
-	else {
-		return false;
-	}
+ImageBrightener::ImageBrightener(const std::shared_ptr<Image>& inputImage) : m_inputImage(inputImage) {
```

Comment:
Do not pass a `shared_ptr` by reference. It loses its ability of reference counting

Reply:
Yes right, done